modules/metering-upgrade-verify.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-upgrading-metering.adoc
+
+[id="metering-upgrade-verify_{context}"]
+= Verifying Metering Upgrade
+
+You can verify the metering upgrade by performing any of the following:
+
+*  Check the Metering Operator ClusterServiceVersion (CSV) for the new metering version through either the web console or by using the `oc` CLI.
+**  Web console:
++
+.Procedure
++
+***  Navigate to *Operators* -> *Installed Operators* in the metering namespace.
+***  Select the Metering Operator to view the *Operator Details* page.
+***  Check the *Installed Version* for the upgraded metering version. The *Starting Version* shows the metering version prior to upgrading.
++
+**  Using the `oc` CLI:
++
+.Procedure
++
+***  Check the Metering Operator CSV:
++
+----
+$ oc get csv
+----
++
+In the following example, the 4.5 Metering Operator upgrade is successful and replaces the 4.4 metering installation:
++
+----
+NAME                                        DISPLAY                  VERSION              REPLACES                               PHASE
+elasticsearch-operator.4.5.0-202006161654   Elasticsearch Operator   4.5.0-202006161654                                          Succeeded
+metering-operator.4.5.0-202006161654        Metering                 4.5.0-202006161654   metering-operator.4.4.0-202005252114   Succeeded
+----
+
+*  Check that all required Pods in the metering namespace are created through either the web console or by using the `oc` CLI.
++
+[NOTE]
+====
+Many Pods rely on other components to function before they themselves can be considered ready. Some Pods may restart if other Pods take too long to start, this is okay and can be expected during the Metering Operator upgrade.
+====
+**  Web console:
++
+.Procedure
++
+***  Navigate to *Workloads* -> *Pods* in the metering namespace and verify that Pods are being created. This can take several minutes after upgrading the metering stack.
++
+**  Using the `oc` CLI:
++
+.Procedure
++
+***  Check that all required Pods in the `openshift-metering` namespace are created:
++
+----
+$ oc -n openshift-metering get pods
+----
++
+The output shows that all Pods are created in the `Ready` column:
++
+----
+NAME                                  READY   STATUS    RESTARTS   AGE
+hive-metastore-0                      2/2     Running   0          3m28s
+hive-server-0                         3/3     Running   0          3m28s
+metering-operator-68dd64cfb6-2k7d9    2/2     Running   0          5m17s
+presto-coordinator-0                  2/2     Running   0          3m9s
+reporting-operator-5588964bf8-x2tkn   2/2     Running   0          2m40s
+----
+
+*  Verify that the `ReportDataSources` are importing new data, indicated by a valid timestamp in the `NEWEST METRIC` column (this may take a few minutes). Filter out the "-raw" `ReportDataSources`, which do not import data:
```

Comment:
IBM Style Guide suggests:
- Try to avoid parentheses. You could just make this a standalone sentence: "This might take a few minutes."
- Use "might" instead of "may", which could imply granting permission.